widgets/src/lib/interfaces/Libraries.ts

```diff
@@ -189,6 +189,13 @@ nlp = spacy.load("${nameWithoutNamespace(model.modelId)}")
 import ${nameWithoutNamespace(model.modelId)}
 nlp = ${nameWithoutNamespace(model.modelId)}.load()`;
 
+const speechbrainAudioClassification = (model: ModelData) =>
+`from speechbrain.pretrained import EncoderClassifier
+
+model = EncoderClassifier.from_hparams(
+  "${model.modelId}"
+)`;
+
```

Comment:
Nit: as in other speechbrain examples [here](https://github.com/huggingface/huggingface_hub/blob/b32ccaa4ab32d5b5ca40273d8a9f321332953de6/widgets/src/lib/interfaces/Libraries.ts#L199-L215),

does it need to follow the format:
```py
from speechbrain.pretrained import EncoderClassifier
audioclassification_model = EncoderClassifier.from_hparams("${model.modelId}") 
audioclassification_model.method("file.wav")
```

Reply:
Done